frontend/src/scenes/billing/billingProductLogic.ts

```diff
@@ -167,5 +193,25 @@ export const billingProductLogic = kea<billingProductLogicType>([
                     DEFAULT_BILLING_LIMIT
             )
         },
+        reportSurveyShown: ({ surveyID, productType }) => {
+            posthog.getActiveMatchingSurveys((surveys) => {
```

Comment:
It seems like you want to use posthog-surveys just for the result visualisations, rather than, say, using the UI to change the question, or target some specific people.

In this case, I don't think you need to check `posthog.getActiveMatchingSurveys` at all: just use the survey ID to send the events to track. You're not using any info from the returned survey anywhere.

With this, even if you make accidental changes to the survey in the UI, nothing changes here.

Reply:
we realized this on friday 😄 changes incoming today